src/Encoding.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Devpark\Transfers24;
+
+/**
+ * Class Encoding.
+ */
+class Encoding extends CodeTranslate
+{
+    const ISO     = 'ISO-8859-2';
+    const UTF     = 'UTF-8';
+    const WINDOWS = 'Windows-1250';
```

Comment:
Why not WINDOWS_1250 ?

Reply:
I have used short encoding names, because Przelewy24 does not accept any other encodings (https://przelewy24.pl/storage/app/media/pobierz/Instalacja/przelewy24_dokumentacja_3.2.pdf, page 6)

Reply:
Yes, but for regular user it's unclear what is UTF or ISO